src/actions/ScanActions.js

```diff
@@ -8,8 +8,8 @@ import URL from 'url-parse'
 
 import { selectWalletForExchange } from '../actions/CryptoExchangeActions.js'
 import { ButtonsModal } from '../components/modals/ButtonsModal.js'
+import { ConfirmContinueModal } from '../components/modals/ConfirmContinueModal.js'
 import { paymentProtocolUriReceived } from '../components/modals/paymentProtocolUriReceived.js'
-import { shouldContinueLegacy } from '../components/modals/shouldContinueLegacy.js'
```

Comment:
Technically shouldn't be removed yet as it's still used in this file as of this commit

Reply:
Correct, I'll fix it.